ci-operator/step-registry/cri-o/integration/cri-o-integration-workflow.yaml

```diff
@@ -3,14 +3,15 @@ workflow:
   steps:
     cluster_profile: equinix-ocp-metal
     pre:
-      - ref: baremetalds-packet-setup
+      - ref: gcp-provision-vpc
```

Comment:
The testing is using "cluster_profile: equinix-ocp-metal", here steps are using gcp ones, is that correct?

Reply:
oops! fixed!